linux_os/guide/system/software/sudo/sudo_restrict_privilege_elevation_to_authorized/rule.yml

```diff
@@ -18,11 +21,18 @@ severity: medium
 
 identifiers:
     cce@sle15: CCE-85712-8
+    cce@rhel7: CCE-83421-8
+    cce@rhel8: CCE-83422-6
 
 references:
     nist: CM-6(b),CM-6(iv)
     disa@sle15: CCI-000366
-    stig@sle15: SLES-15-020101
+    stigid@sle15: SLES-15-020101
+    disa@rhel7: CCI-000366
+    stigid@rhel7: CCI-000366
+    srg: SRG-OS-000480-GPOS-00227
+    disa@rhel8: CCI-000366
```

Comment:
I was discussing with @ggbecker that the `disa` CCI reference is product independent, much like the `srg`.
Reference: https://public.cyber.mil/stigs/cci/

What do you think of removing the product specification?

Reply:
No problem, I will fix it. Just note that we have about 100 cases of disa@something in the content. That's why I was not sure and I rather took this approach.

Reply:
I'll also chime in on this subject since I've seen this recently... in the docs https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#rules

we don't mention the ability to use such a naming convention for `disa` refs, so how did this come to fruition? The only reference we allow such a convention by our docs is the `stigid@product`..

Reply:
I guess every reference can have a product qualifier, the build system does not prevent this. And that probably happened when someone is contributing and wants to make sure that it doesn't affect other products unintentionally.